org-roam-bibtex.el

```diff
@@ -120,7 +120,10 @@ See `orb-edit-notes' for details."
           (const :tag "No" nil))
   :group 'org-roam-bibtex)
 
-(defcustom orb-preformat-keywords '(("citekey" . "=key="))
+(defcustom orb-preformat-keywords
+  '("citekey" "type" "pdf?" "note?"
+    "author" "editor"
+    "author-abbrev" "editor-abbrev" "author-or-editor-abbrev")
```

Comment:
Just to be clear, `author-or-editor-abbrev` doesn't exist yet, but we're working on this in tmalsburg/helm-bibtex/pull/321.

Reply:
Ok.